shopfloor/services/zone_picking.py

```diff
@@ -249,9 +242,7 @@ def _data_for_location(self, zone_location, picking_type, location):
         }
 
     def _zone_lines(self, zones):
-        return self.env["stock.move.line"].search(
-            self._find_location_move_lines_domain(zones)
-        )
+        return self._find_location_move_lines(zones)
```

Comment:
Can't we do the following 
```suggestion
        return self.search_move_line.search_move_lines_by_location(zones)
```
and remove `_find_location_move_lines` method (which seems also used at different places but could be replaced by the component)? Even the `_zone_lines` method could be removed at this stage.

Reply:
I left it on purpose: if you want to completely customize the behavior at scenario level.

Reply:
@sebalix beside this the rest is good?

Reply:
gut!